e2e/schema/node-no-network.json

```diff
@@ -345,13 +345,11 @@
   "required": [
     "allocatableCpuCores",
     "allocatableEphemeralStorageBytes",
-    "allocatableHugepages1Gi",
```

Comment:
Why do we need to remove those? Some explanation would be nice.

Reply:
This metric was not present on my local minikube environment, i didn't spent any time looking at the root cause , I will check it.

Reply:
I mean, perhaps we should have some alternative test for this metric then?

Reply:
i was not able to launch minikube locally having this resource 😞 to document it somewhere , do you have any suggestions? I fear if that we do not remove from required others might suffer the same as I locally.

Reply:
I removed this commit , i will remove this metrics locally